docs/source/dev_guide.rst

```diff
@@ -31,14 +31,15 @@ Running tests
 
 Run tests using pytest_.
 
-First activate the ``thunderbird`` Conda environment and install ``pytest``.
+First activate the ``thunderbird`` Python environment and install ``pytest``.
 
 .. code-block:: console
 
-   $ source activate thunderbird
-   $ pip install -r requirements_dev.txt  # if not already installed
+   $ python3 -m venv venv
+   $ source venv/bin/activate
+   (venv)$ pip install -r requirements_dev.txt  # if not already installed
    OR
-   $ make develop
+   (venv)$ make develop
 
 Run quick tests (skip slow and online):
 
```

Comment:
[Line48](https://github.com/pacificclimate/thunderbird/pull/66/files#diff-334dcd038b07c3f543c49f630ed7df85R48) This does not necessarily have to be changed, but I believe we do not have any test that is `slow`.

Reply:
You are right.  I'm hoping to add those in the future.